.goreleaser.yml

```diff
@@ -1,76 +1,95 @@
 # GoReleaser config
 
+---
 before:
   hooks:
     - go mod tidy
 
 builds:
-- id: pop_darwin
-  binary: soda
-  main: soda/main.go
-  goos:
-    - darwin
-  goarch:
-    - amd64
-  env:
-    - CGO_ENABLED=1
-    - CC=o64-clang
-    - CXX=o64-clang++
-  flags:
-    - -tags
-    - sqlite
+  - id: pop_darwin
+    binary: soda
+    main: soda/main.go
+    goos:
+      - darwin
+    goarch:
+      - amd64
+    ldflags:
+      - -s
+    env:
+      - CGO_ENABLED=1
+      - CC=/osxcross/target/bin/o64-clang
+      - CXX=/osxcross/target/bin/o64-clang++
+    flags:
+      - -tags
+      - sqlite
 
-- id: pop_linux
-  binary: soda
-  main: soda/main.go
-  env:
-    - CGO_ENABLED=1
-  flags:
-    - -tags
-    - sqlite
-  goos:
-    - linux
-  goarch:
-    - amd64
-    - 386
+  - id: pop_darwin_arm64
+    binary: soda
+    main: soda/main.go
+    goos:
+      - darwin
+    goarch:
+      - arm64
+    ldflags:
+      - -s
+    env:
+      - CGO_ENABLED=1
+      - CC=/osxcross/target/bin/oa64-clang
+      - CXX=/osxcross/target/bin/oa64-clang++
```

Comment:
I separated out pop_darwin and pop_darwin_arm64 as darwin_arm64 suggests oa64-clang/oa64-clang++ on https://github.com/goreleaser/goreleaser-cross#supported-toolchainsplatforms.